advanced-game-design-team/Artifacts/Raycast Collisions/raycast.cpp

```diff
@@ -0,0 +1,81 @@
+#include "./include/types.h"
+#include <algorithm>
+#include <limits>
+
+ray create_ray(point_2d origin, vector_2d direction) {
+    return {origin, unit_vector(direction)};
```

Comment:
You've once again taught me more modern C++ stuff here with list initialization, but unfortunately for us dumbasses it means there's one more thing we need to look up when we see it because I doubt this has been added to SIT102. For our sake adding even just `// list initialization` so that we have a keyword to look up would be really handy.